### ISSUE TITLE
Source questions from OpenTDB

### ISSUE THREAD

**Issue by TingPing** [martinszeltins/who-wants-to-be-a-millionaire#3]
There is a free service called Open Trivia Database: https://opentdb.com/

This would allow dynamically generating new questions each run.

Admittedly many of the questions are not up to the quality of the real thing but I believe its one of the few freely licensed collections of trivia.

**Comment by martinszeltins** [comment]
> There is a free service called Open Trivia Database: [opentdb.com](https://opentdb.com/)
> 
> This would allow dynamically generating new questions each run.
> 
> Admittedly many of the questions are not up to the quality of the real thing but I believe its one of the few freely licensed collections of trivia.

I like this idea. I see that they have 3 categories - "easy", "medium" and "hard". But for a game like this each question should be progressively more difficult. So something like 15 difficulty levels?

**Comment by TingPing** [comment]
Yeah, I was thinking to request 3 lists for every game and spread it out. Hard to match the real thing.
